pyqtgraph/parametertree/parameterTypes/action.py

```diff
@@ -29,26 +68,24 @@ def treeWidgetChanged(self):
         tree.setItemWidget(self, 0, self.layoutWidget)
 
     def titleChanged(self):
-        self.button.setText(self.param.title())
         self.setSizeHint(0, self.button.sizeHint())
 
-    def optsChanged(self, param, opts):
-        ParameterItem.optsChanged(self, param, opts)
-
-        if 'enabled' in opts:
-            self.button.setEnabled(opts['enabled'])
-
-        if 'tip' in opts:
-            self.button.setToolTip(opts['tip'])
-
-    def buttonClicked(self):
-        self.param.activate()
-
 
 class ActionParameter(Parameter):
     """Used for displaying a button within the tree.
 
     ``sigActivated(self)`` is emitted when the button is clicked.
+
+    ============== ============================================================
```

Comment:
Would you mind changing this docstring to numpy style docstring?  We are hoping to migrate the rest of the repo to that, but it may be a while.

Also I think it should be 

```
:class:`QIcon`  and :class`QKeySequence`
```

Our docs are now smart enough to link to the upstream Qt docs for Qt types like that 👍🏻